jobs/telegraf-agent/spec

```diff
@@ -16,6 +16,12 @@ properties:
     default: {}
     example: {datacenter: dc1}
 
+  agent_conf:
```

Comment:
🤔 should we instead add an explicit property or properties for what we're trying to expose?

alternatively, could we just call this `agent` so it matches the config scheme of the other properties? (pretty much 1:1 to the .ini values they set)